packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -22,7 +22,7 @@ declare module '@generated/site-metadata' {
   import type {DocusaurusSiteMetadata} from '@docusaurus/types';
 
   const siteMetadata: DocusaurusSiteMetadata;
-  export default siteMetadata;
+  export = siteMetadata;
```

Comment:
not sure to understand what's the advantage of this syntax?

Reply:
This is about how you're able to import it. Because we have `esModuleInterop` there's not a visible change. I forgot why I did this change but at least now it's "more correct". `export = X` is for `module.exports = X` and `export default X` is for `module.exports.default = X`.